src/ProjectTemplates/test/Templates.Tests/template-baselines.json

```diff
@@ -1500,6 +1500,22 @@
         "wwwroot/css/open-iconic/font/fonts/open-iconic.woff"
       ]
     },
+    "Empty": {
+      "Template": "blazorserver-empty",
+      "Arguments": "new blazorserver-empty",
+      "Files": [
+        "App.razor",
+        "appsettings.Development.json",
+        "appsettings.json",
+        "MainLayout.razor",
+        "Program.cs",
+        "_Imports.razor",
+        "Pages/Index.razor",
+        "Pages/_Host.cshtml",
+        "Properties/launchSettings.json",
```

Comment:
Can we just add a second test here with `exclude-launch-settings`?

Reply:
Yes, let me add that right now.